zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -222,16 +222,55 @@ table.dataTable.table-condensed .sorting_desc:after {
   cursor: pointer;
 }
 
-.dropdown-menu .shortcut-icon{
+.dropdown-menu .shortcut-icon {
   padding-right: 7px;
   float: left;
 }
 
-.dropdown-menu .shortcut-keys{
+.dropdown-menu .shortcut-keys {
   float: right;
   color: #999;
 }
 
+.modal-dialog .table-scroll {
+  max-height: 70vh;
+  min-width: 160px;
+  overflow: auto;
+  overflow-y: scroll;
+}
+
+.shortcut-modal-header {
+  min-height: 16.428571429px;
+  padding: 15px;
+  border-bottom: 1px solid #9cb4c5;
+  background-color: #3071a9;
```

Comment:
What about adding `border-top-left-radius` and `*-right-radius` here since bottom has `border-radius` in modal content.

![image](https://cloud.githubusercontent.com/assets/4968473/25282156/a51b08d8-26ea-11e7-8c14-7a9575327871.png)

![image](https://cloud.githubusercontent.com/assets/4968473/25282190/b49e75ec-26ea-11e7-8dcf-a1cc388b745a.png)